.github/workflows/backend-ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: Backend build
+
+on:
+    push:
+        branches:
+            - main
+            - '[0-9]+.[0-9]+'
+    pull_request: ~
+
+jobs:
+    cs-fix:
+        name: Run code style check
+        runs-on: "ubuntu-22.04"
+        strategy:
+            matrix:
+                php:
+                    - '8.1'
```

Comment:
can we run this on 8.3?

Reply:
Actually it seems we can: https://github.com/ibexa/phpstan/pull/1/commits/a2113b9ea9f3aed1105bdf072b1a56533b12a26a.